### DISCUSSION TITLE
Generated XML files cause crawling errors in Google Search Console. What's the best way to handle this?

### DISCUSSION THREAD

**Discussion by bugok** [jpanther/congo#913] [General]
I have a congo-based site: https://www.noamlerner.com. The code for the site is here: https://github.com/bugok/blog
The site is deployed using cloudflare pages. 

I'm tracking my site in Google Search Console - and I see that there are many failures related to crawling xml files: 

![Screenshot 2024-07-15 at 9 53 51](https://github.com/user-attachments/assets/f5f8ee83-14d2-491e-bca3-4eeb4addf633)

I'm not familiar enough with congo to understand if the xml files should be generated or not. Are they used to render the content of the site? If they are needed - what's the desired solution here to make Google Search Console happy? Ignore those xml files in robots.txt? Some other way? 

Thanks.

**Comment by wolfspyre** [comment]
could be any number of things as i’m not terribly familiar with goog’s search console.. what’s it saying the problem is?

i spoze it could be a mime type thing, or they don’t conform completely to a schema or they’re not excluded from robots.txt or they’re not sparkly enough :)

point being there could be a few causes and without more information on the actual problem it’s hard to say much

**Comment by bugok** [comment]
Here's one example I get for the 'crawled - not currently indexed' error:

![Screenshot 2024-07-15 at 10 45 18](https://github.com/user-attachments/assets/8d6daed9-6f57-40c5-86a9-daa66cd295aa)

When I added a 'Disallow' statement for the path it changed to this: 
![Screenshot 2024-07-15 at 10 45 41](https://github.com/user-attachments/assets/d4da5605-d9e8-4a8e-be3d-0d330a9021f4)

Maybe this is the solution? 

In the meantime - I updated my robots.txt to exclude xml files: https://github.com/bugok/blog/blob/main/layouts/robots.txt

@wolfspyre : What do you think?
